umap/static/umap/content.css

```diff
@@ -248,7 +248,7 @@ input[type="submit"],
 ul.umap-autocomplete {
     position: absolute;
     background-color: white;
-    z-index: 1010;
+    z-index: 10100;
```

Comment:
Let's talk about a strategy here 😅 

```css
:root {
  --level-base: auto;
  --level-panels: 10;
  --level-autocomplete: 20;
  --level-errors: 30;
  and so on
}
```
and when we need more:

```css
.autocomplete-input {
  z-index: calc(var(--level-autocomplete) + 1);
}
```

These are dumb examples but you hopefully see what I mean.

Reply:
Let's do that in a dedicated PR! :)